### DISCUSSION TITLE
pagFile.setTimeStretchMode(PAGTimeStretchMode.Scale)无效

### DISCUSSION THREAD

**Discussion by Taot7** [Tencent/libpag#2368] [Q&A]
`pagFile.setTimeStretchMode(PAGTimeStretchMode.Scale)
            pagFile.setDuration(60000000)`
我想将一段30s的动画设定为0.5倍速播放，但是这段代码实际运行结果为前30秒无动画，后30秒正常播放动画

**Comment by domchen** [comment]
有完整代码吗？
